llama.cpp

```diff
@@ -1361,22 +1361,26 @@ static bool llama_model_load(
 
 // evaluate the transformer
 //
-//   - lctx:         llama context
-//   - tokens:       new batch of tokens to process
-//   - n_past:       the context size so far
-//   - n_threads:    number of threads to use
-//   - cgraph_fname: filename of the exported computation graph
+//   - lctx:      llama context
+//   - tokens:    new batch of tokens to process
+//   - n_tokens   number of tokens
+//   - embd       embeddings input
+//   - n_past:    the context size so far
+//   - n_threads: number of threads to use
 //
 static bool llama_eval_internal(
-        llama_context &  lctx,
-    const llama_token *  tokens,
-            const int    n_tokens,
-            const int    n_past,
-            const int    n_threads,
+         llama_context & lctx,
+     const llama_token * tokens,
+             const int   n_tokens,
+           const float * embd,
+             const int   n_past,
+             const int   n_threads,
```

Comment:
Much better. Minor nitpick - lets change the order of the args:

```suggestion
         llama_context & lctx,
     const llama_token * tokens,
           const float * embd,
             const int   n_tokens,
             const int   n_past,
             const int   n_threads,
```

Reply:
OK. I have changed it.